client/src/components/RoomDetails/index.jsx

```diff
@@ -1,67 +1,188 @@
-    "https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQU8wtCeoV6WwgF7AxMZ7Pb1WbRtWcSnaXY0g&s",
-    "https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcRD6Kn6DuellFmvWWhJAmHXTedcQTEyJP8wgg&s.jpg",
-  ];
-
-  const changeSlide = (direction) => {
-    setSlideIndex((prevIndex) => (prevIndex + direction + images.length) % images.length);
+ useEffect(() => {
+  const fetchFeaturedRooms = async () => {
+    try {
+      const response = await fetch(`${process.env.REACT_APP_SERVER_URI}/rooms/${id}`);
+      const data = await response.json();
+      setroomDetailsData(data);
+    } catch (error) {
+      console.error('Error fetching featured rooms:', error);
+    } finally {
+      setLoading(false);
+    }
   };
 
-  return (
-    <div>
-      <header className="header">
-        <h1>Room Details - {roomType}</h1>
-      </header>
+  fetchFeaturedRooms();
+ }, [id]);
+
+ const images = [
+   "https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcS-9adA4eaOlSpe_LJEKzxGJjigLJtjGs8ATg&s",
+   "https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQU8wtCeoV6WwgF7AxMZ7Pb1WbRtWcSnaXY0g&s",
+   "https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcRD6Kn6DuellFmvWWhJAmHXTedcQTEyJP8wgg&s",
+ ];
+
+ const changeSlide = (direction) => {
+   setSlideIndex((prevIndex) => (prevIndex + direction + images.length) % images.length);
+ };
+
+ const roomDetails = {
+   name: "Luxury Ocean Suite",
+   description: "Experience unparalleled luxury in our meticulously designed ocean-view suite. This spacious accommodation offers the perfect blend of comfort and sophistication, with breathtaking views and world-class amenities.",
+   price: 200,
+   size: "45 m²",
+   occupancy: "2 Adults, 1 Child",
+   bedType: "King Size",
+   view: "Ocean View",
+   amenities: [
+     { icon: "🛏️", name: "King-sized Bed", description: "Premium mattress and linens" },
+     { icon: "📶", name: "High-speed WiFi", description: "Complimentary high-speed internet" },
+     { icon: "🏊‍♂️", name: "Pool Access", description: "Exclusive infinity pool access" },
+     { icon: "🍽️", name: "Room Service", description: "24/7 in-room dining" },
+     { icon: "🚿", name: "Luxury Bathroom", description: "Rain shower and premium amenities" },
+     { icon: "❄️", name: "Climate Control", description: "Individual temperature control" },
+     { icon: "📺", name: "Entertainment", description: "55-inch Smart TV" },
+     { icon: "☕", name: "Mini Bar", description: "Complimentary refreshments" }
+   ]
+ };
+
+ const handleSubmit = async (e) => {
+  e.preventDefault();
+  const stripe = await loadStripe('pk_test_51QH8UhDVCApEbmXH0V05yN5isrK6roHA6ix4SyzpaxA1cWrfhPJSkWCI3fuTImeCVCgfjBCBkOn2i1ysZBov9vc500rvVtvsjX');
+  const response = await fetch(`${process.env.REACT_APP_SERVER_URI}/payment`, {
+      method: 'POST',
+      headers: {
+          'Content-Type': 'application/json'
+      },
+      body: JSON.stringify({ amount: 1000 }) 

```

Comment:
Why are we sending default value? Update this line

Reply:
I was testing the stripe api and forgot to remove the static value.